Gemfile.lock

```diff
@@ -25,9 +25,6 @@ GEM
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.0.3)
-    active_link_to (1.0.5)
-      actionpack
```

Comment:
I take it actionpack and addressable are also no longer needed? They aren't dependencies for anything else?

Reply:
I assume so since these changes were generated from bundle install